src/elements/content-picker/Content.tsx

```diff
@@ -1,3 +1,9 @@
+/**
+ * @flow
+ * @file File picker header and list component
+ * @author Box
+ */
+
```

Comment:
Please delete this comment, you don't need the header comment at the top of the file